machine.go

```diff
@@ -148,6 +148,15 @@ IgnoreSIGPIPE=no
 SendSIGHUP=yes
 `
 
+const localfs = `
+[Unit]
+Description=Local File Systems
+Documentation=man:systemd.special(7)
+DefaultDependencies=no
+Conflicts=shutdown.target
+After=local-fs-pre.target
```

Comment:
That should really have a `FailureAction=poweroff-immediate` or somesuch; if mounting failes we want to bail from the fakemachine rather then muddling forwards with the invariants broken

Reply:
Actually it  would  be better not to overwrite local-fs.target but replace emergency.service by that calls poweroff immediately that would "fix" all cases where fakemachine would accidentally go into an emergency shell